pkg/pods/manifest.go

```diff
@@ -167,20 +177,39 @@ func (m manifestBuilder) SetConfig(config map[interface{}]interface{}) error {
 	return nil
 }
 
-func (manifest *manifest) GetStatusPort() int {
-	return manifest.StatusPort
+func (manifest *manifest) GetStatusHTTP() bool {
+	if manifest.StatusHTTP {
+		return true
+	}
+	return manifest.Status.HTTP
 }
 
-func (manifest *manifest) SetStatusPort(port int) {
-	manifest.StatusPort = port
+func (manifest *manifest) SetStatusHTTP(statusHTTP bool) {
+	manifest.StatusHTTP = false
+	manifest.Status.HTTP = statusHTTP
 }
 
-func (manifest *manifest) GetStatusHTTP() bool {
-	return manifest.StatusHTTP
+func (manifest *manifest) GetStatusPath() string {
+	if manifest.Status.Path != "" {
+		return path.Join("/", manifest.Status.Path)
```

Comment:
What do you mean? This is meant to normalize the URL so that it has one and only one leading slash.